MatrixSDK/MXSession.h

```diff
@@ -357,6 +358,11 @@ FOUNDATION_EXPORT NSString *const kMXSessionNoRoomTag;
  */
 @property (nonatomic, readonly) MXRestClient *matrixRestClient;
 
+/**
+ The identity service used to handle Matrix identity server requests.
```

Comment:
This is old code. We do not have `NS_ASSUME_NONNULL_BEGIN` but we can say it can be nil  in the comments.